### ISSUE TITLE
Single Node Kubernetes instllation referece link broken

### ISSUE THREAD

**Issue by kumarashit** [scality/Zenko#1070]
# General Support Information

GitHub issues are reserved for actionable bug reports (including documentation
issues), and feature requests.

**All questions** regarding configuration, use cases, performance, community,
events, setup and usage recommendations, among other things should be asked on
the **[Zenko Forum](http://forum.zenko.io/)**.

_Questions opened as GitHub issues will be summarily closed and moved to the
[Zenko Forum](http://forum.zenko.io/)._

## Avoid Duplication

Before reporting a new issue or requesting a feature, please search the issue
list for this repository to ensure the issue has not already been opened (Use
the search bar and select "Issues" on the left pane after searching). If the
issue has already been raised, do not open a new issue. Add a comment to the
existing issue instead.

--------------------------------------------------------------------------------

## Bug Report Information

If you are reporting a bug, provide the information described here. If you are
requesting a new feature, delete this section (to the next line).

### Description

Link for Single node Kubernetes installation gives 404 error

### Steps to Reproduce the Issue

From the landing page check README and click on sing-node kubernetes installation https://github.com/scality/Zenko/blob/development/2.0/docs/minikube.md

### Actual Results
404

### Expected Results

Open the page for installation instructions

### Additional Information

Let us know how your deployed example is configured. Tell us your:

- Node.js version
- Docker version
- npm version
- Distribution/OS

Include anything else you think will help us resolve the issue.

--------------------------------------------------------------------------------

## Feature Requests

If you are requesting a new feature, delete the previous section (Bug Report
Information) and provide the information described here.

### Proposal

Describe the feature you want.

### Current Behavior

Tell us what the program s doing now that you want to change.

### Desired Behavior

Tell us what you want the program to do.

### Use Cases

Please provide use cases for changing the program's current behavior. Tell us
the starting state for the desired new feature (i.e., what screen or process
step you want to start from), the steps you'd like to follow (or avoid!), and
the results you want.

### Additional Information

To serve you better, we'd like to know a little more about you (if you don't
mind our asking...)

- Is this request for your company?
  - If it is, for which company?
  - Is your company using any Scality Enterprise Edition products (RING, Zenko
    EE)?
- Are you willing to contribute this feature yourself?
- What is your position or title?
- How did you hear about us?

**Comment by rahulreddy** [comment]
@kumarashit Thanks for pointing the bad link. Minikube is not maintained anymore and will be removed from references. Are you trying to install Zenko in a Kubernetes environment?

**Comment by kumarashit** [comment]
@rahulreddy yes trying to install zenko in kubernetes ennv. Please point me to deployment documentation

**Comment by kumarashit** [comment]
i realized that the "testing zenko" both links are broken.
is metalk8s is the way for on-prem deployment?

**Comment by rahulreddy** [comment]
@kumarashit Our 2.0 is still under development. I recommend trying 1.2.1 release, here are the instructions https://zenko.readthedocs.io/en/1.2.1/installation/index.html
The install instructions for volume setup has steps for MetalK8s, you can choose to use MetalK8s for on-prem or go with any vanilla K8s cluster. If you do run in into issues setting it up please let me know or post in forum.zenko.io, I can get you some help.

**Comment by kumarashit** [comment]
thanks @rahulreddy I will try it out and let you know.
